.github/workflows/test.yml

```diff
@@ -1,6 +1,11 @@
 name: Build & Test CI
 
-on: push
+on:
+  push:
+  workflow_call:
+    secrets:
+      CHROMATIC_PROJECT_TOKEN:
+        required: true
```

Comment:
What does adding the `required` here do? Just error out sooner in ci?

Reply:
i think so. before we were reusing this config to run all tests for releases. Andrew and I found that was redundant given how our lifecycle is set up now. 

We could remove this, but if we ever need to reuse this workflow in another workflow, we are set up to fail if the keys aren't passed over